src/SDKs/IotCentral/Management.IotCentral/Microsoft.Azure.Management.IotCentral.csproj

```diff
@@ -0,0 +1,27 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <!-- Please do not move/edit code below this line -->
+  <Import Project="$([MSBuild]::GetPathOfFileAbove('AzSdk.reference.props'))" />
+  <!-- Please do not move/edit code below this line -->
+
+  <PropertyGroup>
+    <PackageId>Microsoft.Azure.Management.IotCentral</PackageId>
+    <Description>Provides management capabilities for Microsoft Azure IotCentral.</Description>
+    <AssemblyTitle>Microsoft Azure IotCentral Management</AssemblyTitle>
+    <AssemblyName>Microsoft.Azure.Management.IotCentral</AssemblyName>
+    <Version>1.0.0-preview</Version>
```

Comment:
If you are planning to release a preview version, the versioning must start from `0.9.0-preview`

Reply:
I missed this earlier but we also need `Assembly.cs` files similar to this for both the projects with the assembly versions properly set.

Reply:
I've added Assembly.cs and changed the csproj version to `0.9.0-preview`

![image](https://user-images.githubusercontent.com/6381138/41621790-943d3538-73c2-11e8-8a6a-08815483412d.png)